src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1299,14 +1299,18 @@ public static SqlConnectionStringBuilder CreateConnectionStringBuilder(Connectio
 
                 connectionBuilder.EnclaveAttestationUrl = connectionDetails.EnclaveAttestationUrl;
             }
-            if (connectionDetails.Encrypt.HasValue)
+            if (connectionDetails.Encrypt != null)

```

Comment:
So this is going to require updating the code in the other services too, right? (any anything else that creates its own connections)

Reply:
All consuming libraries would need to update the same in order to work with TDS 8 using MDS 5.0. 
If you continue to set these to true/false, it won't break apps, but TDS 8 support would be missing.